tests/DCPS/Compiler/xcdr/xcdr.idl

```diff
@@ -76,21 +85,45 @@ enum UnionDisc {
   E_SHORT_FIELD,
   E_LONG_FIELD,
   E_OCTET_FIELD,
-  E_LONG_LONG_FIELD
+  E_LONG_LONG_FIELD,
+  E_ADDITIONAL_FIELD
 };
 
 @mutable
 @OpenDDS::data_representation(XCDR1)
 @OpenDDS::data_representation(XCDR2)
 union MutableXcdr12Union switch (UnionDisc) {
-case E_SHORT_FIELD:
-  short short_field;
-case E_LONG_FIELD:
-  long long_field;
-case E_OCTET_FIELD:
-  octet octet_field;
-case E_LONG_LONG_FIELD:
-  long long long_long_field;
+  COMMON_BRANCHES
+};
+
+@final
+struct NoDataRepFinalStruct {
```

Comment:
Eventually all the data representation annotations and any duplicate types that causes are going to be removed from this file.

Reply:
This will be done in task XTYPE-118.